dcrwallet.go

```diff
@@ -289,7 +290,13 @@ func rpcClientConnectLoop(legacyRPCServer *legacyrpc.Server, loader *wallet.Load
 				legacyRPCServer.SetChainServer(chainClient)
 			}
 			if cfg.EnableTicketBuyer {
-				startTicketPurchase(w, chainClient.Client, nil, cfg.tbCfg)
+				pm, err = startTicketPurchase(w, chainClient.Client, nil, cfg.tbCfg)
+				if err != nil {
+					log.Errorf("Unable to start ticket buyer: %v", err)
+					return
+				}
+				pm.Start()
+				loader.SetWaitFn(pm.WaitForShutdown)
```

Comment:
I don't see Stop called anywhere except below in this same function.  What stops the purchase manager when the wallet is stopped from e.g. gRPC?